core/src/main/java/org/apache/accumulo/core/classloader/DefaultContextClassLoaderFactory.java

```diff
@@ -102,20 +101,4 @@ public ClassLoader getClassLoader(String contextName) {
     return org.apache.accumulo.start.classloader.vfs.AccumuloVFSClassLoader
         .getContextClassLoader(contextName);
   }
-
```

Comment:
With your change in ClassLoaderUtil above, which essentially reverts the changes in #3678, I think you can remove the contextName checks in getClassLoader (lines 98-100). They were added in #3678. I wonder if it would be easier to revert #3678 and then apply the changes in ShellServerIT, PropUtil, SystemPropUtil, and ClassLoaderUtil.